lib/features/children/overview/pages/models/top_up_success_uimodel.dart

```diff
@@ -0,0 +1,9 @@
+class TopUpSuccessUIModel {
```

Comment:
🎉

Reply:
Good work in implementing the UIModel. The only thing I would change is I would remove the "onClickButton" here as it isn't really "UI" but rather "logic". I would pass it as a parameter in the TopUpSuccessPage itself. That way when we use UIModels in our cubits the cubits don't need to know/ define the onClick.